### ISSUE TITLE
Nuget feed v3 only

### ISSUE THREAD

**Issue by xavierzwirtz** [fsprojects/Paket#1143]
Working on a private nuget feed implementing the v3 nuget 3 spec. If I set the feed in `paket.dependencies` to 
```
source https://myfeed.org/api/v3
```

Paket uses the v2 methods to attempt to load the data. I would like to not have to implement the v2 api in my feed, its rather complicated for what I need to do. Any way to have Paket use the v3 api methods for a specific url? Maybe with an argument to source that hard codes the api version.
```
source https://myfeed.org/api/v3 nugetv3
```

**Comment by forki** [comment]
Currently paket accepts only v2 urls and automatically tries to compute the
v3 url and then tests this url.

You could look into this v3 url discovery (in NugetV3.fs) to make it work
for your url.

**Comment by forki** [comment]
(you are the first person that tells me there are nuget v3 servers that
don't implement v3)

**Comment by xavierzwirtz** [comment]
There isnt one yet ;)
On 10/15/2015 5:41:57 PM, Steffen Forkmann <notifications@github.com> wrote:
(you are the first person that tells me there are nuget v3 servers that
don't implement v3)

**Comment by xavierzwirtz** [comment]
I am working on a pr that will make Paket support a nuget feed that does not implement the v2 OData api.

I think the source parameter needs to support explicitly setting the api version, ala `source https://myfeed.org/api/v3 nugetv3`

Would you have any objections to that?

**Comment by xavierzwirtz** [comment]
For backwards compatibility any source that does not have a version explicitly set is assumed to be `nugetv2`. Only issue I could see with this is if some one was using a custom feed url that has a space in it that is not url encoded to `%20`.

**Comment by xavierzwirtz** [comment]
A lot of the tests specify a source with only a starting quote, not an ending quote:
`source "http://nuget.org/api/v2`

Does that functionality need to stay? Its making adding version to the parse a pain.

**Comment by forki** [comment]
I don't really understand why you need to change the parser.
I think the only interesting part is https://github.com/fsprojects/Paket/blob/master/src/Paket.Core/NuGetV3.fs#L30

**Comment by xavierzwirtz** [comment]
I am being an idiot. I was going to make it where you had to explicitly say that a source supported V3, no reason to do that though. :see_no_evil:

**Comment by xavierzwirtz** [comment]
Implemented by #1205
